ui/src/ngm-app.ts

```diff
@@ -125,13 +121,16 @@ export class NgmApp extends LitElementI18n {
   private queryManager: QueryManager | undefined;
   private waitForViewLoading = false;
   private resolutionScaleRemoveCallback: Event.RemoveCallback | undefined;
+  private readonly disclaimer: CoreModal | null = null;
 
   @consume({context: clientConfigContext})
   accessor clientConfig!: ClientConfig;
 
   constructor() {
     super();
 
+    this.disclaimer = CoreModal.open({isPersistent: true, size: 'large', hasNoPadding: true}, html`<ngm-tracking-consent @confirm="${(event) => this.onTrackingAllowedChanged(event)}"></ngm-tracking-consent>`);
```

Comment:
Split this across multiple lines, + pass `onTrackingAllowedChanged` directly.
```suggestion
    this.disclaimer = CoreModal.open({isPersistent: true, size: 'large', hasNoPadding: true}, html`
      <ngm-tracking-consent
        @confirm="${this.onTrackingAllowedChanged}"
      ></ngm-tracking-consent>
    `);
```

Reply:
When i pass `onTrackingAllowedChanged ` directly, i get an nerror when clicking the accept-button, that `showNavigationHint` is not a function